src/flash/com/longtailvideo/jwplayer/media/RTMPMediaProvider.as

```diff
@@ -680,6 +680,12 @@ public class RTMPMediaProvider extends MediaProvider {
             case 'NetStream.Seek.Notify':
                 sendMediaEvent(MediaEvent.JWPLAYER_MEDIA_SEEKED);
                 break;
+            case 'NetStream.Buffer.Empty':
+                sendBufferEvent(0);
+                break;
+            case 'NetStream.Buffer.Full':
+                sendBufferEvent(100);
```

Comment:
`sendBufferEvent(100)` means 100% of the timeline is buffered. 100 might be ok for live streams, but otherwise we want to calculate buffered percent when duration is known:
```as
var bufferPercent:Number = 100;
if (_item.duration > 0) {
    bufferPercent = 100 * (_stream.time + _stream.bufferLength) / _item.duration;
}
```